cmd/translate/main.go

```diff
@@ -20,7 +20,7 @@ import (
 	"google.golang.org/grpc"
 	"google.golang.org/grpc/credentials/insecure"
 
-	pb "go.expect.digital/translate/pkg/server/translate/v1"
+	translatev1 "go.expect.digital/translate/pkg/pb/translate/v1"
```

Comment:
is v1 at the end of translate necessary?

Reply:
<img width="987" alt="image" src="https://user-images.githubusercontent.com/69954124/225591863-65385e95-b28c-4984-878b-5b5a8adbc614.png">

Thats how creator of linter suggested to do